src/main/context-handler.ts

```diff
@@ -52,13 +57,7 @@ export class ContextHandler {
       return await provider.getPrometheusService(apiClient);
     });
     const resolvedPrometheusServices = await Promise.all(prometheusPromises);
-    const service = resolvedPrometheusServices.filter(n => n)[0];
-    return service || {
-      id: "lens",
-      namespace: "lens-metrics",
-      service: "prometheus",
-      port: 80
-    };
+    return resolvedPrometheusServices.filter(n => n)[0];
```

Comment:
Do we want to return `undefined` if there are no services?

Reply:
yes